testsuites/syncgateway/functional/tests/test_continuous.py

```diff
@@ -26,9 +29,20 @@ def test_continuous_changes_parametrized(params_from_base_test_setup, sg_conf_na
 
     cluster_conf = params_from_base_test_setup["cluster_config"]
     mode = params_from_base_test_setup["mode"]
+    ssl_enabled = params_from_base_test_setup["ssl_enabled"]
 
     sg_conf = sync_gateway_config_path_for_mode(sg_conf_name, mode)
 
+    # Skip the test if ssl disabled as it cannot run without port using http protocol
```

Comment:
Tests should be able to run using the couchbase:// protocol without a port if ssl is disabled.

Reply:
Actually , in this case, I use http if ssl is disabled , so I want to make sure that test gets skipped if http is used without port.